src/main/java/gyro/google/storage/BucketAccessControlProjectTeam.java

```diff
@@ -0,0 +1,69 @@
+/*
+ * Copyright 2019, Perfect Sense, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package gyro.google.storage;
+
+import com.google.api.services.storage.model.BucketAccessControl.ProjectTeam;
+import gyro.core.resource.Diffable;
+import gyro.core.resource.Updatable;
+import gyro.core.validation.ValidStrings;
+import gyro.google.Copyable;
+
+/**
+ * The project team associated with the entity.
+ */
+public class BucketAccessControlProjectTeam extends Diffable implements Copyable<ProjectTeam> {
+
+    private String projectNumber;
+    private String team;
+
+    /**
+     * The project number.
+     */
+    @Updatable
```

Comment:
Must have an explicit primary key, otherwise changes to this updatable field triggers a create/delete flow

Reply:
I am removing this complex type. You can't actually insert via the API.